From e9f66ed8d7dd1ab7fd4a3bd6bc1864ce182d9de8 Mon Sep 17 00:00:00 2001
Message-Id: <e9f66ed8d7dd1ab7fd4a3bd6bc1864ce182d9de8.1361535607.git.minovotn@redhat.com>
From: Michal Novotny <minovotn@redhat.com>
Date: Fri, 22 Feb 2013 13:18:41 +0100
Subject: [PATCH] Revert "block: fix block tray status"

Reverting as asked by Ademar.

This reverts commit 8a5181c3fa0b1efb69153485808afcb612d0aa60.

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/block.c b/block.c
index 03e5e65..1f75c37 100644
--- a/block.c
+++ b/block.c
@@ -964,10 +964,9 @@ void bdrv_close(BlockDriverState *bs)
             bdrv_close(bs->file);
         }
 
-    }
-
-    if (!runstate_check(RUN_STATE_INMIGRATE)) {
-        bdrv_dev_change_media_cb(bs, false);
+        if (!runstate_check(RUN_STATE_INMIGRATE)) {
+            bdrv_dev_change_media_cb(bs, false);
+        }
     }
 }
 
-- 
1.7.11.7

